spec/controllers/characters_controller_spec.rb

```diff
@@ -405,7 +405,7 @@
       expect(character.characters_galleries.first).not_to be_added_by_group
 
       login_as(user)
-      put :update, params: { id: character.id, character: {ungrouped_gallery_ids: ['']} }
+      put :update, params: { id: character.id, character: {ungrouped_gallery_ids: [''], gallery_group_ids: [group.id]} }
```

Comment:
What's up with this?

Reply:
Oh I remember! Because the thing wasn't there, it was taking that as "this character should be assigned to have zero gallery groups", and then it wasn't persisting galleries that were tethered by gallery groups _because there were also no gallery groups_. So this is now necessary for the group to stay attached. Shouldn't matter in regular use, because this would be sent with the edit page as usual; just an edge case of not sending 100% of normal parameters in a test.